src/gridstack.ts

```diff
@@ -1854,6 +1854,10 @@ export class GridStack {
       this.opts.marginLeft = data.height;
       delete this.opts.margin;
     }
+    
+    //without this.opts.margin, getMargin method will return undefined, and affect resize behavior
+    if(!this.opts.margin) this.opts.margin = Math.round((this.opts.marginTop + this.opts.marginBottom) / 2);

```

Comment:
this isn't the right fix because margin=0 is a valid value. Thanks for ringing this issue, will fix it the proper way next...